lib/course_planner/notifications.ex

```diff
@@ -16,4 +17,18 @@ defmodule CoursePlanner.Notifications do
   def to(%Notification{} = notification, %User{} = user),
     do: %{notification | user: user}
 
+  def send_all_notifications do
+    Timex.today()
```

Comment:
i mean by limiting the notification between 1-7 days it is good to give the coordinator the power to stop all the notifications.
e.g. they first fill everything in the application and then activate the notification